pymc3/distributions/distribution.py

```diff
@@ -230,16 +230,65 @@ def draw_values(params, point=None):
     """
     # Distribution parameters may be nodes which have named node-inputs
     # specified in the point. Need to find the node-inputs to replace them.
-    givens = {}
+    
+    # Issue #2900 describes a situation in which, the named node-inputs
```

Comment:
This information is too detailed here. It is better to put into the release note.